### ISSUE TITLE
"Select all" option and sorting button for task lists

### ISSUE THREAD

**Issue by Gaploid** [Azure/BatchLabs#809]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report
[x] feature request
```

**Description**

- If it possible please add "select all" ability in task lists to do a bulk delete for example. 

- The sorting in tables will be awesome too. For example to sort by dates (created, completed and etc)

**Instruction to repro the problem if applicable**

**Comment by timotheeguerin** [comment]
* Sorting needs to be done client side as the batch service doesn't support that yet. That means you'd only sort the displayed items which might be quite confusing.

* Select all: Do you mean as a shortcut to delete all task on the screen or all task in a job(even the one not loaded) you can also already shift click to select multiple tasks, jobs,etc. to delete them in bulk

**Comment by Gaploid** [comment]
- Yep sorting should be server side. 

- If there is an ability to select all including not loaded yet it will be awesome.

**Comment by timotheeguerin** [comment]
Yeah server side sorting has been a quite requested feature but is probably coming in the near future.
For the select all that could be possible.
